corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -50,6 +51,18 @@ hqDefine('registration/js/new_user.ko', function () {
             if (_private.isAbPhoneNumber) {
                 _kissmetrics.track.event("Phone Number Field Filled Out");
             }
+
+            var appcuesEvent = "Assigned user to Appcues test",
+                appcuesData = {
+                    'Appcues test': data.appcuesAbTest,
+                };
+
+            _appcues.identify(data.email, appcuesData);
+            _appcues.trackEvent(appcuesEvent, appcuesData);
+
+            _kissmetrics.identify(data.email);
```

Comment:
I think this could also be passed from appcues directly. Or is this necessary for another reason?

Reply:
You mean turning on appcues' integration with kiss? Yeah, I think that'd be equivalent. I'll ask Marshall if it matters to him. (I'm a bit skeptical of appcues integrations because the product is so new...but we have plenty of issues of our own with the analytics we send out to multiple services not staying in sync, so neither way seems obviously better to me.)